### DISCUSSION TITLE
r/PolishFiltersTeam is a private community

### DISCUSSION THREAD

**Discussion by xxcriticxx** [MajkiIT/polish-ads-filter#19914] [General]
r/PolishFiltersTeam is a private community

**Comment by hawkeye116477** [comment]
Zgadza się. Wstęp jest tylko dla VIPów :smile:.

**Reply by xxcriticxx** [reply]
jak dobrze ze ja nie jestem vip

**Reply by hawkeye116477** [reply]
Aby zostać VIPem należy dać 5$ dla Ojca Dyrektora albo wziąć udział w wyzwaniu - zrobić jak najwięcej zgłoszeń w ciągu dnia w repo @MajkiIT (należy pokonać obecnego mistrza) :smiley:

**Reply by xxcriticxx** [reply]
mamy juz w tym czempiona

**Reply by hawkeye116477** [reply]
No właśnie już to pisałem, trzeba przebić jego wynik.

**Reply by xxcriticxx** [reply]
niedługo felix powróci z nowym nickiem

**Comment by hawkeye116477** [comment]
A tak na serio, to mało było gości na tym subie, a nie można było skasować, więc jest ustawione jako prywatne. 
Zapomnieliśmy tylko o opisie, można było zmienić to fakt, a teraz już za późno, wszyscy opuścili suba (Krystian chyba nawet konto skasował), więc trzeba byłoby pisać do admina, by przydzielił dostęp.
